apicapi/apic_client.py

```diff
@@ -31,13 +31,14 @@
 
 APIC_CODE_FORBIDDEN = str(requests.codes.forbidden)
 APIC_CODE_SSL_ERROR = str(requests.codes.gateway_timeout)
+APIC_CODE_UNAUTHORIZED = str(requests.codes.unauthorized)
 
 FALLBACK_EXCEPTIONS = (rexc.ConnectionError, rexc.Timeout,
                        rexc.TooManyRedirects, rexc.InvalidURL)
 SLEEP_TIME = 0.03
 SLEEP_ON_FULL_QUEUE = 1
 
-REFRESH_CODES = [APIC_CODE_FORBIDDEN, ]
+REFRESH_CODES = (APIC_CODE_FORBIDDEN, APIC_CODE_UNAUTHORIZED, )
```

Comment:
overall looks good but is there a reason to change it from [] to () here?

Reply:
tuples are immutable, lists are not. Just safer.